### PR TITLE
Add Katib metric strategies info

### PR DESCRIPTION
**Pull request by andreyvelich**
Fixes: https://github.com/kubeflow/katib/issues/1310.
Blocked by: https://github.com/kubeflow/website/pull/2312.

I added doc about `metricStrategies` in Objective section.

/assign @johnugeorge @gaocegege 

/cc @8bitmp3

### CODE REVIEW

**Comment by andreyvelich** [comment]
This PR is ready.
/cc @gaocegege @8bitmp3 @johnugeorge

File: content/en/docs/components/katib/experiment.md [hunk 1]
```diff
@@ -64,14 +64,45 @@ These are the fields in the experiment configuration spec:
   how the hyperparameters work with the model.
   Katib records the value of the best `objectiveMetricName` metric (maximized
   or minimized based on `type`) and the corresponding hyperparameter set
-  in Experiment's `.status.currentOptimalTrial.parameterAssignments`.
+  in experiment's `.status.currentOptimalTrial.parameterAssignments`.
```
Comment (8bitmp3) [inline content/en/docs/components/katib/experiment.md:67]:
```suggestion
  in the experiment's `.status.currentOptimalTrial.parameterAssignments`.
```

File: content/en/docs/components/katib/experiment.md [hunk 2]
```diff
@@ -64,14 +64,45 @@ These are the fields in the experiment configuration spec:
...
 
+  The default way to calculate the experiment's objective is:
+
+  - When the objective `type` is `maximize`, Katib compares all maximum
+    metric values.
+
+  - When the objective `type` is `minimize`, Katib compares all minimum
+    metric values.
+
+  You are able to change the default mechanism. For that, define
```
Comment (8bitmp3) [inline content/en/docs/components/katib/experiment.md:83]:
```suggestion
  To change the default settings, define
```

File: content/en/docs/components/katib/experiment.md [hunk 3]
```diff
@@ -64,14 +64,45 @@ These are the fields in the experiment configuration spec:
...
+  `metricStrategies` with various rules (`min`, `max` or `latest`) to extract
+  values for each metric from the experiment's `objectiveMetricName` and
+  `additionalMetricNames`. The experiment's objective value is calculated in
+  accordance with the selected strategy.
+
+  For example, if you set the below parameters in your experiment:
```
Comment (8bitmp3) [inline content/en/docs/components/katib/experiment.md:89]:
```suggestion
  For example, you can set the parameters in your experiment as follows:
```

File: content/en/docs/components/katib/experiment.md [hunk 4]
```diff
@@ -64,14 +64,45 @@ These are the fields in the experiment configuration spec:
...
+  . . .
+  objectiveMetricName: accuracy
+  type: maximize
+  metricStrategies:
+    - name: accuracy
+      value: latest
+  . . .
+  ```
+
+  Katib controller is searching for the best maximum from the all latest
```
Comment (8bitmp3) [inline content/en/docs/components/katib/experiment.md:101]:
```suggestion
  where the Katib controller is searching for the best maximum from the all latest
```

**Review by 8bitmp3 — CHANGES_REQUESTED** [review]
PTAL 👍

**Review by andreyvelich — COMMENTED** [review]
Thank you for the review @8bitmp3.
I made changes.

**Review by gaocegege — COMMENTED** [review]
LGTM :+1:
/lgtm

**Pull request merged**